<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>950</width>
    <height>620</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>950</width>
    <height>620</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QListWidget" name="vystup_pre_dialog">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>300</width>
      <height>551</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>300</width>
      <height>0</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="obrazok_2">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>10</y>
      <width>300</width>
      <height>291</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="obrazok_1">
    <property name="geometry">
     <rect>
      <x>316</x>
      <y>11</y>
      <width>300</width>
      <height>291</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTabWidget" name="nastavenia">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>320</y>
      <width>621</width>
      <height>241</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>620</width>
      <height>240</height>
     </size>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Rounded</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="elideMode">
     <enum>Qt::ElideNone</enum>
    </property>
    <widget class="QWidget" name="segmentacia">
     <attribute name="title">
      <string>Segment√°cia</string>
     </attribute>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>10</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>10 &lt;= W &lt;= 35</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spin_W">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>40</y>
        <width>81</width>
        <height>22</height>
       </rect>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="minimum">
       <number>10</number>
      </property>
      <property name="maximum">
       <number>35</number>
      </property>
      <property name="value">
       <number>14</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="spin_T">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>40</y>
        <width>81</width>
        <height>22</height>
       </rect>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="minimum">
       <number>80</number>
      </property>
      <property name="maximum">
       <number>255</number>
      </property>
      <property name="value">
       <number>200</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>10</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>80 &lt;= T &lt;= 255</string>
      </property>
     </widget>
     <widget class="QPushButton" name="button_zmen">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>100</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Zmen</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>950</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="movable">
    <bool>true</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
